seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/sink/PaimonSinkWriter.java

```diff
@@ -224,4 +250,17 @@ public void close() throws IOException {
             committables.clear();
         }
     }
+
+    private boolean waitCompaction() {
+        CoreOptions.ChangelogProducer changelogProducer =
+                this.table.coreOptions().changelogProducer();
+        return changelogProducer == CoreOptions.ChangelogProducer.LOOKUP
+                || changelogProducer == CoreOptions.ChangelogProducer.FULL_COMPACTION;
+    }
+
+    public static String[] splitPaths(@Nonnull String separatedPaths) {
```

Comment:
This method is already in org.apache.paimon.disk.IOManagerImpl#splitPaths. You can directly import it.